packages/databyss-ui/editor/EditorInline.js

```diff
@@ -24,6 +24,8 @@ const EditorInline = React.forwardRef(
         ref={ref}
         borderRadius={5}
         p="tiny"
+        pr="0"
+        ml="-3px"
```

Comment:
use `pxValue` for cross-platform
better yet, add a new size called `negativeTiny` so that they always match